utils/test/prepdata.sh

```diff
@@ -3,7 +3,16 @@ set -e
 
 project_source_dir=$1
 
+ncgen -o rads_adt_3a_2021181.nc4 ${project_source_dir}/testdata/rads_adt_3a_2021181.cdl
+ncgen -o rads_adt_3b_2021181.nc4 ${project_source_dir}/testdata/rads_adt_3b_2021181.cdl
+ncgen -o rads_adt_3a_2021182.nc4 ${project_source_dir}/testdata/rads_adt_3a_2021182.cdl
```

Comment:
You're going to hate me @apchoiCMD ! 
Let's just commit 2 .cdl files. Sorry, I should have thought about this earlier.

Reply:
@guillaumevernieres only 2 files of cdl? like "rads_adt_3a_2021181.nc4" and "rads_adt_3b_2021181.nc4"?

Reply:
Yes, sounds good @apchoiCMD

Reply:
@guillaumevernieres  Updated prepdata.sh and removed testdata files as well